xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-macro/src/main/resources/ApplicationResources.properties

```diff
@@ -77,6 +77,8 @@ livedata.selection.infoBar.selectedCount={0} entries selected
 livedata.selection.infoBar.allSelected=All entries selected
 livedata.selection.infoBar.allSelectedBut=All entries selected but {0}
 
+livedata.pagination.label={0} live data page info
```

Comment:
I'm not sure about this string. First, is "page info" really how you call this? Not rather something like pagination/pagination control? Second, while the term "live data" might be well-known to regular editors of XWiki, I don't think it is used anywhere in the UI. Therefore, somebody visiting the wiki might not be aware what "live data" is. Third, I'm not sure it is a good idea to start with the id, which might be a technical name and might not be properly pronounced by a TTS engine.

I'm not sure we have a non-technical name how we call "live data" in the UI. So what about something like `Pagination controls for {0}`? Or is it a good practice not to start labels with the same string so users can more quickly differentiate between different labels? I think it could also make sense to add a comment for translators what the parameter content is.

Reply:
> Third, I'm not sure it is a good idea to start with the id, which might be a technical name and might not be properly pronounced by a TTS engine.

The id is defined by the creator of the table. From what I saw in the few LiveTables we have by default in XWiki, it was properly set and quite easy to read. E.g. menu, childrenIndex, ...
To make things cleaner we could create a `label` parameter for the LiveTables, but I think using ids is good enough here.

>I'm not sure we have a non-technical name how we call "live data" in the UI. So what about something like Pagination controls for {0}?

We do use it in the UI only for class names :+1: 
`Pagination controls for {0}` is a way better value than before, thanks! :)
I think I'd make it a bit more precise so that the user knows that they are going to read through a table entry representation of an object, and not a table with the object itself:
`Pagination controls for the {0} table`

> Or is it a good practice not to start labels with the same string so users can more quickly differentiate between different labels? I think it could also make sense to add a comment for translators what the parameter content is.

I don't think this label will be read out too often, so it's okay to have the "specific" part of the label at the end: 
it would have been less okay if we had this read out on every entry in the table. In most cases one page will only contain one table, and even in niche cases, I don't see it being more than two or three tables.